files/collect_api_events.rb

```diff
@@ -11,44 +11,47 @@
 require_relative 'util/processor'
 require_relative 'util/logger'
 
+confdir     = ARGV[0] || '/etc/puppetlabs/puppet/common_events'
+modulepaths = ARGV[1] || '/etc/puppetlabs/code/environments/production/modules:/etc/puppetlabs/code/environments/production/site:/etc/puppetlabs/code/modules:/opt/puppetlabs/puppet/modules'
+statedir    = ARGV[2] || '/etc/puppetlabs/puppet/common_events'
+
 def main(confdir, _modulepaths, statedir)
-  log = CommonEvents::Logger.new('/tmp/common_events.log')
+  log      = CommonEvents::Logger.new('/tmp/common_events.log', settings['loglevel'])
```

Comment:
Are we adding the logger level stuff in this PR?

Reply:
Ohh, haha. Nope. This is left over scratch notes typing. Good catch.